### PR TITLE
Added edit paragraph actions and commands

### PR DESCRIPTION
**Pull request by AndreasArvidsson**

### CODE REVIEW

File: core/edit/edit_paragraph.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        if is_line_empty():
+            return
+        # Search for start of paragraph
+        actions.edit.extend_paragraph_start()
+        actions.edit.left()
+        # Extend to end of paragraph
+        actions.edit.extend_paragraph_end()
+
+    def extend_paragraph_start():
+        extend_paragraph_start()
```
Comment (nriley) [inline core/edit/edit_paragraph.py:27]: You might want to explain why you don't use the default Talon supplied action.

File: core/edit/edit.talon [hunk 2]
```diff
@@ -161,6 +165,7 @@ paste match: edit.paste_match_style()
 (pace | paste) line: user.paste_line()
```
Comment (nriley) [inline core/edit/edit.talon:165]: Can we add an optional "to" to make the use of these clearer?

Comment (AndreasArvidsson) [reply core/edit/edit.talon]: done

File: core/edit/edit_paragraph.py (hunk 1 above)
Comment (AndreasArvidsson) [reply core/edit/edit_paragraph.py]: done

**Comment by AndreasArvidsson** [comment]
@nriley Ready for review again :)

**Review by nriley — APPROVED** [review]

**Pull request merged**